ebean-core-type/src/main/java/io/ebean/core/type/DataBinder.java

```diff
@@ -174,4 +174,9 @@ public interface DataBinder {
    */
   String popJson();
 
+  /**
+   * Returns the last bound object (e.g. for BindValidation). Note for InputStreams you'll get an InputStreamInfo.
```

Comment:
Add docs (and/or method name, e.g. 'popLastLengthObject' ), that only Strings & InputStreams are returned